tests/scripts/pgocheck/pefile/pefile.py

```diff
@@ -2528,7 +2522,7 @@ def parse_directory_bound_imports(self, rva, size):
                 if name_str:
                     invalid_chars = [
                         c for c in bytearray(name_str) if
-                            chr(c) not in string.printable]
+                            unichr(c) not in string.printable]
```

Comment:
unichr and chr can have slightly different semantics depending on how Python 2 is configured.  We should make sure that the Python that we are using is configured to use UCS4; as this will ensure that we are getting the same semantics as using the chr function from Python 3.